doc/source/Manual.rst

```diff
@@ -1859,7 +1859,7 @@ If you have Git_ installed on your development system,
 you can use it together with pip
 to install the latest version of |PyInstaller| directly::
 
-    pip install -e git://github.com/pyinstaller/pyinstaller.git#egg=PyInstaller
+    pip install https://github.com/pyinstaller/pyinstaller/archive/develop.zip
```

Comment:
Ah yes. I think it's simpler to just mention the `-e` way, as it doesn't hurt anything. Right?